package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+
+<package>
```

Comment:
Just a suggestion: When using format 2 (`<package format="2">`), there is a `<depend>`-tag which combines build- and run-depend, i.e. you don't need to list them twice.

Reply:
I did not know that, cool +1